code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -205,6 +205,24 @@ datum
 			reagent_state = LIQUID
 			color = "#B97A57"
 
+		fishwater
+			name = "Fish Water"
+			id = "fishwater"
+			description = "Smelly water from a fish tank. Gross!"
+			reagent_state = LIQUID
+			color = "#757547"
+
+			on_mob_life(var/mob/living/M as mob)
+				if(!M) M = holder.my_atom
+				M << "Oh god, why did you drink that?"

```

Comment:
Good point, that message would get spammy if some idiot decided to try chugging the entire wall aquarium...

The chance to vomit from it each cycle was intended though. You might swallow it fine, but then have trouble actually keeping it down, just like the gallon of milk challenge.